packages/frontend/src/config/buyNearConfig.js

```diff
@@ -6,13 +6,21 @@ import monoPay from '../components/buy/assets/monoPay.svg';
 import okex from '../components/buy/assets/okex.svg';
 import payNear from '../components/buy/assets/payNear.svg';
 import rainbow from '../components/buy/assets/rainbow.svg';
+import transakLogo from '../components/buy/assets/transak.svg';
 import utorg from '../components/buy/assets/utorg.svg';
 import { Mixpanel } from '../mixpanel';
 
-export const getPayMethods = ({ accountId, moonPayAvailable, signedMoonPayUrl, utorgPayUrl, ftxPayUrl }) => {
-    return {
+export const getPayMethods = ({
+    accountId,
+    moonPayAvailable,
+    signedMoonPayUrl,
+    utorgPayUrl,
+    transakPayUrl,
+    ftxPayUrl,
+}) => {
+    const paymentMethods = {
         moonPay: {
-            disabled: accountId && !moonPayAvailable,
+            disabled: accountId && moonPayAvailable,
```

Comment:
We want to disable this TopUp method if moonPayAvalaible returns false. Can we revert this back to read `disabled: accountId && !moonPayAvailable`?

Reply:
Indeed! Forgot to revert change after testing